forte/sparse_ci/determinant.hpp

```diff
@@ -108,6 +108,17 @@ template <size_t N> class DeterminantImpl : public BitArray<N> {
             set_beta_bit(p, occupation_b[p]);
     }
 
+    /// Construct the determinant from an occupation vector that
+    /// specifies the alpha and beta strings.  occupation = [Ia,Ib]
+    explicit DeterminantImpl(const BitArray<nbits_half>& Ia, const BitArray<nbits_half>& Ib) {
+        for (size_t n = 0; n < nwords_half; n++) {
```

Comment:
Can this constructor not call `set_str`? The logic here is just a call to that function.

Reply:
Good point!